userspace/libsinsp/filter.cpp

```diff
@@ -344,7 +352,7 @@ Json::Value sinsp_filter_check::rawval_to_json(uint8_t* rawval, const filterchec
 			{
 		 		return (Json::Value::Int64)*(int64_t *)rawval;
 			}
-			else 
+			else
```

Comment:
There are a bunch of space-only changes in this patch. Please try to avoid them.